votor/src/consensus_pool/vote_pool.rs

```diff
@@ -191,6 +211,8 @@ struct Stakes {
     /// A single validator may vote for upto 3 blocks and different validators can vote for different blocks.
     /// Hence, this tracks stake per block id.
     notar_fallback: BTreeMap<Hash, Stake>,
+    /// Stake that has voted genesis.
+    genesis: BTreeMap<Hash, Stake>,
```

Comment:
Just to confirm my understanding - this structure is just accumulating a u64 (`Stake`) and we use other code (below) to ensure we aren't adding duplicate votes:
https://github.com/anza-xyz/agave/blob/d5757e29aa9f40fd75c265ea1f581130a3430536/votor/src/consensus_pool/vote_pool.rs#L283

Reply:
Yep exactly, this just plugs into the existing mechanism to aggregate stake from votes. We specify the certificate threshold and the votes per validator limit which is handled in add vote to throw away duplicate votes and create the certificate.